packages/algob/src/internal/deployer.ts

```diff
@@ -113,8 +112,7 @@ class DeployerBasicMode {
    * @returns {LogicSig} multi signed logic signature from assets/<file_name>.(b)lsig
    */
   async loadMultiSig (name: string, scParams: LogicSigArgs): Promise<LogicSig> {
-    if (name.endsWith(blsigExt))
-      return loadBinaryMultiSig(name);
+    if (name.endsWith(blsigExt)) { return await loadBinaryMultiSig(name); }
```

Comment:
this functions is removed

Reply:
this line was edited automatically when i did `yarn run lint:fix` with latest master pull. 
i think this function is not removed, it can be viewed in master branch

Reply:
I was talking about `loadBinaryMultiSig` - I think @ratik21  removed it.

Reply:
it is moved to other file, but not removed